.github/workflows/deployment.yml

```diff
@@ -72,18 +72,24 @@ jobs:
       environment: ${{ env.ENVIRONMENT }}
       timezone: ${{ env.TIMEZONE }}
     steps:
+      - name: Configure 🔨
+        if:  ${{ !contains(fromJson('["feature", "staging", "prod"]'), github.event.workflow_run.head_branch) }}
+        run: |
+          echo "ENVIRONMENT=dev" >> $GITHUB_ENV
+          echo "ENVIRONMENT=dev" >> $GITHUB_OUTPUT
+
       - name: Environment 🧪
-        run: echo "Environment set to ${{ env.ENVIRONMENT }}"
+        run: echo "Environment has been set to ${{ env.ENVIRONMENT }}"
 
       - name: Timezone 🌐
         run: echo "Timezone set to ${{ env.TIMEZONE }}"
 
-  sql-migrations:
-    name: SQL Migrations 🗄️
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
-    needs: setup
-    runs-on: [self-hosted, DTFS, deployment]
+  database:
+    name: MSSQL 🗄️
+    if: ${{ github.event.workflow_run.conclusion == 'success' && vars.DATABASE == '1' }}
```

Comment:
Could we move this now @abhi-markan ? everyhing is on SQL so we always want to run migrations now

Reply:
This is to avoid running on `prod`

Reply:
Ah I guess if release-1.6 beats us there then we won't want it to run
Also jsut realised- these changes are on deployment.yml when we only care about them on main